extensions/amp-audio/0.1/amp-audio.js

```diff
@@ -58,40 +59,46 @@ export class AmpAudio extends AMP.BaseElement {
 
   /** @override */
   buildCallback() {
+    const viewer = Services.viewerForDoc(this.getAmpDoc());
+    viewer.whenFirstVisible().then(() => {
```

Comment:
Actually, because this element is not `prerenderAllowed`, we don't need to wait for viewer visible. `buildCallback` itself won't be called until visible.

Reply:
Took that out. PTAL?